packages/collector/Collector_Content.php

```diff
@@ -0,0 +1,46 @@
+<?php
+function collector_iterate_directory($path, $prefix, $callback)
+{
+	$handle = opendir($path);
+
+	while($entry = readdir($handle))
+	{
+		if($entry === '.' || $entry === '..' || substr($entry, 0, 4) === '.git')
```

Comment:
Or .svn, or .mercurial etc. I wonder there's any blanket rule we could apply here, like "don't ship hidden directories" 🤔

Reply:
@adamziel How does this look for now? We can iterate on it in the future if the need presents itself:
https://github.com/WordPress/playground-tools/pull/124/files#diff-585e75f8b3fbea54031d37f43d8d2eddf519d49278e0b7b47e37fa7bbf972aabR8

Reply:
@adamziel I think we can add a configuration option for this stuff. Is this something we can iterate on?

Reply:
ec5b36741505c08173e5772ad4ba52a6b75f75e8 adds support for skipping all hidden files and folders. We can add a filter in the future to allow for more granular importing.